docs/reference/errors-and-warnings/NU5001.md

```diff
@@ -0,0 +1,25 @@
+---
+title: NuGet Error NU5001
+description: NU5001 Error code
+author: mishra14
+ms.author: anmishr
+manager: rrelyea
+ms.date: 8/14/2018
+ms.topic: reference
+ms.reviewer: anangaur
+f1_keywords:
+  - NU5001
+---
+
+# NuGet Error NU5001
+<pre>Unable to output resolved nuspec file because it would overwrite the original at 'F:\project\project.nuspec'.</pre>
+
+### Issue
+
+The NuGet pack operation was invoked with the `-InstallPackageToOutputPath` option. But the output path already contained a  nuspec file.
```

Comment:
`option. But the output path already contained a nuspec file` -> `option but the output path already has a nuspec file`